example_publisher/providers/pyth_replicator.py

```diff
@@ -51,14 +50,14 @@ async def _update_loop(self) -> None:
                 symbol = update.product.symbol
 
                 if self._prices.get(symbol) is None:
-                    self._prices[symbol] = [None, None, None]
+                    self._prices[symbol] = (None, None, None)
 
                 if update.aggregate_price_status == PythPriceStatus.TRADING:
-                    self._prices[symbol] = [
```

Comment:
Wonder if self._prices should be `Dict[Id, Price]` here for consistency with `coin_gecko.py`

Reply:
I refactored the whole file and fixed all the type issues.